apps/sms/test/marionette/composer_test.js

```diff
@@ -322,5 +322,13 @@ marionette('Messages Composer', function() {
       assert.deepEqual(contact.name, newMessage.recipients);
       assert.equal(contact.tel[0].value, newMessage.recipientsPhoneNumbers);
     });
+
+    test('should accept semicolon to completes the entered recipient',
+    function() {
+      var separator = ';';
+      newMessage.addNewRecipient(123, separator);
```

Comment:
I created a second test to check the input readiness, so it'll make the error message clearer. So we have:

```js
    suite('Semicolon as separator', function() {
      var separator = ';';

      test('should complete the entered recipients', function() {
        newMessage.addNewRecipient(123, separator);
        assert.lengthOf(newMessage.recipients, 1);
        assert.notInclude(newMessage.recipients[0], separator);
      });

      test('should leave the input ready to accept another recipient',
      function() {
        newMessage.addNewRecipient(123, separator);
        newMessage.addNewRecipient(456, separator);
        assert.lengthOf(newMessage.recipients, 2);
      });
    });
```

@azasypkin Are you fine with it?